label_studio/ml_models/serializers.py

```diff
@@ -20,3 +22,36 @@ class Meta:
         model = ModelInterface
         fields = '__all__'
         read_only_fields = ['created_by', 'created_at', 'updated_at']
+
+
+class ThirdPartyModelVersionSerializer(serializers.ModelSerializer):
+    created_by = UserSimpleSerializer(default=CreatedByFromContext(), help_text='User who created Dataset')
+
+    class Meta:
+        model = ThirdPartyModelVersion
+        fields = '__all__'
+        read_only_fields = ['created_by', 'created_at', 'updated_at']
+
+    def validate(self, data):
```

Comment:
Does the check that the parent model exists happen automagically by django, w/ the foreign key relationship? (didn't see anything explicitly set up in the model/migration, but could be missing something there too)

Reply:
Yes that is handled thanks to the ForeignKey field, and captured in one of the tests I added